src/box/lua/net_box.lua

```diff
@@ -390,6 +391,24 @@ local function new_sm(uri, opts)
                     })
                 end
             end
+            if not opts.fetch_schema and features.space_and_index_names then
+                remote.space = setmetatable({_remote = remote}, {
+                    __index = function(self, key)
+                        local space = {id = key, name = key,
+                                       _remote = self._remote,
+                                       _format_cdata =
```

Comment:
`_format_cdata` is never used.

Reply:
It is used in requests (see space and index object metatabels).

Reply:
Missed that, sorry.

Reply:
Actually, `nil` format is ok:
https://github.com/tarantool/tarantool/blob/a27d958d060ae2725ed8df6c1a4cc2a51d776233/src/box/lua/net_box.c#L2255-L2258

Dropped this field.